src/utils/message.rs

```diff
@@ -3,16 +3,36 @@ use fvm_shared::message::Message;
 use fvm_shared::{address::Address, econ::TokenAmount, METHOD_SEND};
 
 pub fn message_transfer(from: Address, to: Address, value: TokenAmount) -> Message {
+    create_message(
+        from,
+        to,
+        value,
+        0,
+        TokenAmount::from_atto(0),
+        TokenAmount::from_atto(0),
+        0,
+    )
+}
+
+pub fn create_message(
```

Comment:
Let's have some docs for this as well. The name could also be more informative; `message` is ambiguous.